packages/vats/src/core/startWalletFactory.js

```diff
@@ -15,35 +14,67 @@ const { details: X } = assert;
 // eslint-disable-next-line no-unused-vars
 const startFactoryInstance = (zoe, inst) => E(zoe).startInstance(inst);
 
+const StableUnit = BigInt(10 ** Stable.displayInfo.decimalPlaces);
```

Comment:
wonder if units should go in with their Brand def (i.e. tokens.js in this case)

Reply:
Perhaps, if there are other static uses of it... but a quick search doesn't find any other occurrences of `10 ** Stable.displayInfo.decimalPlaces`.